content/articles/2017-12-29-dbplyr-1-2.Rmd

```diff
@@ -0,0 +1,129 @@
+---
+title: dbplyr 1.2.0
+author: Edgar Ruiz
+date: '2017-12-29'
+slug: dbplyr-1-2
+categories:
+  - package
+tags: [package, tidyverse, databases]
+photo:
+  url: https://unsplash.com/photos/y7rGTFyOzxc
+  author: Giulia Bertelli
+---
+
+```{r setup, include = FALSE}
+library(dbplyr)
+library(dplyr)
+library(DBI)
+library(tibble)
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+We are very excited to announce that [dbplyr 1.2.0](https://cran.rstudio.com/web/packages/dbplyr/index.html)  is now available on CRAN!  
+
+The dbplyr package is the database backend for dplyr. It interacts with databases directly by translating the dplyr verbs into SQL queries. Other advantages of this approach are:
+
+<img src = "/images/dbplyr_1-2/interact.png" width = 400, align = "right">
+
+
+**Run data exploration routines over all of the data**, instead of subsets or samples.
+
+**Use the SQL Engine to run the data transformations**. In effect, dplyr pushes the computation to the database.
+
+**Collect into R only a targeted dataset.**
+
+**All of your code is in R! **  Because dplyr communicates directly with the database, there is no need to alternate between languages or tools.
+
+dbplyr is part of an ecosystem of packages meant to help analysts access and analyze data in databases. To learn more, visit our website dedicated to *all things R and databases*: http://db.rstudio.com/overview
+
+Install the latest version of dbplyr with:
+```{r, eval =  FALSE}
+install.packages("dbplyr")
+```
+
+## What's new?
+
+Here are some highlights of new features and fixes found in this release.  To see the full details please refer to the [release notes](https://github.com/tidyverse/dbplyr/blob/master/NEWS.md#dbplyr-120):
+
+- New custom translation for **Microsoft Access** and [Teradata](http://db.rstudio.com/databases/teradata/) 
+
+- [Amazon Redshift](http://db.rstudio.com/databases/redshift/) connections now point to the PostgreSQL translation.
+
+- Adds support for **two new database R packages**.  These new packages are fully [DBI](http://db.rstudio.com/dbi/)-compliant and tested with [DBItest](https://cran.r-project.org/web/packages/DBItest/index.html). We recommend to use these instead of older packages:
+    1. [RMariaDB](https://cran.r-project.org/web/packages/RMariaDB/index.html), use in favor of RMySQL
```

Comment:
We need to get pkgdown sites for these packages too. That would be something you could help out with next year, if you were interested.

Reply:
Yeah! I'll be happy to help out with them